xwiki-platform-core/xwiki-platform-web/src/main/webapp/templates/displayer_page.vm

```diff
@@ -1 +1,23 @@
-#template('displayer_dblist.vm')
+#if ($type == 'edit' || $type == 'search')
+  #if ($field.getProperty('displayType').value == 'input' && $field.getProperty('picker').value == 1)
+    #xpropertySuggestInputDisplayer($field $prefix $name $value)
+  #else
+    $doc.displayEdit($field, $prefix, $object)
+  #end
+#elseif ($type == 'view')
+  #set ($value = $doc.displayView($field, $prefix, $object))
```

Comment:
You already have ``$value``. I don't understand why you need to call ``displayView``.

Reply:
What do you mean I already have `$value`? What I wanted was to use `displayView` only if the method didn't return a page reference (e;g. Main.WebHome). In the other case I wanted to render the title of the page reference.